app/assets/images/stembolt_logo.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="248px" height="45px" viewBox="0 0 248 45" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 40.3 (33839) - http://www.bohemiancoding.com/sketch -->
```

Comment:
Here's an awesome tool that will help cut the size of this file down: https://jakearchibald.github.io/svgomg/